docs/data-sources/cloud_provider_aws_account.md

```diff
@@ -17,9 +17,18 @@ data "grafana_cloud_stack" "test" {
   slug = "gcloudstacktest"
```

Comment:
all files in `docs/` are autogenerated using `go generate`, they are built from the resource schemas + description strings added in `internal/resources/cloudprovider` files.